packages/ord-connect/src/hooks/useSignMessage.tsx

```diff
@@ -1,20 +1,47 @@
 import { useCallback, useState } from "react";
+import { AddressFormat } from "@ordzaar/ordit-sdk";
 
 import signMessage from "../lib/signMessage.ts";
-import { useOrdConnect } from "../providers/OrdConnectProvider";
+import {
+  BiAddressFormat,
+  BiAddressString,
+  useOrdConnect,
+} from "../providers/OrdConnectProvider";
+
+function getAddressFormat(
+  address: string,
+  walletAddresses: BiAddressString,
+  walletFormats: BiAddressFormat,
+): AddressFormat {
+  const keys = Object.keys(walletAddresses) as (keyof BiAddressString)[];
+  for (let i = 0; i < keys.length; i += 1) {
+    if (address === walletAddresses[keys[i]]) {
+      return walletFormats[keys[i]]!;
+    }
+  }
+
+  throw new Error("Address does not exist");
+}
```

Comment:
won't need this function once the address is checked

Reply:
updated